Tests/SwiftLintFrameworkTests/VerticalWhitespaceClosingBracesRuleTests.swift

```diff
@@ -0,0 +1,75 @@
+@testable import SwiftLintFramework
+import XCTest
+
+class VerticalWhitespaceClosingBracesRuleTests: XCTestCase {
+    func testVerticalWhitespaceClosingBracesWithDefaultConfiguration() {
+        verifyRule(VerticalWhitespaceClosingBracesRule.description)
+    }
+
+    private let nonTrivialLinesNonTriggeringExamples: [Example] = [
+        Example("""
+        if bool1 {
+          // do something
+          // do something
+
+        } else if bool2 {
+          // do something
+          // do something
+          // do something
+
+        } else {
+          // do something
+          // do something
+        }
+        """)
+    ]
+
+    private let nonTrivialLinesViolatingToValidExamples: [Example: Example] = [
+        Example("""
+        print([
+          1
+        ↓
+        ])
+        """):
+            Example("""
+                    print([
+                      1
+                    ])
+                    """),
+        Example("""
+        print([foo {
+          var sum = 0
+          for i in 1...5 { sum += i }
+          return sum
+
+        }, foo {
+          var mul = 1
+          for i in 1...5 { mul *= i }
+          return mul
+        ↓
+        }])
+        """):
+            Example("""
+            print([foo {
+              var sum = 0
+              for i in 1...5 { sum += i }
+              return sum
+
+            }, foo {
+              var mul = 1
+              for i in 1...5 { mul *= i }
+              return mul
+            }])
+            """)
+    ]
+
+    func testMultilineArgumentsWithOnlyEnforceBeforeTrivialLines() {
+        let description = VerticalWhitespaceClosingBracesRule.description
+            .with(nonTriggeringExamples: (nonTrivialLinesNonTriggeringExamples +
+                                          nonTrivialLinesViolatingToValidExamples.values))
+            .with(triggeringExamples: Array(nonTrivialLinesViolatingToValidExamples.keys))
+            .with(corrections: nonTrivialLinesViolatingToValidExamples.removingViolationMarkers())
```

Comment:
No need to remove the violation markers.

Reply:
Removed.

Reply:
@SimplyDanny Just tried to run the tests.
They - both my own and the auto-generated ones - fail without the `.removingViolationMarkers()`.
What did you mean by this?

Reply:
In triggering examples, the markers show the position where the violation triggers. This is different in correction tests. In there, the locations mark the positions of applied fixes. Both can be different but are often the same. So to fix the test, the positions of the markers would either have to be changed or you remove them (as you did) in correction tests.

That said, you may ignore my comment. Sorry for the noise.